packages/commonwealth/client/scripts/views/pages/manage_community/chain_metadata_rows.tsx

```diff
@@ -23,6 +23,12 @@ import {
   setChainCategories,
 } from './helpers';
 import { CWLabel } from '../../components/component_kit/cw_label';
+import {
+  Action,
+  addPermission,
+  isPermitted,
+  removePermission,
+} from '../../../../../../common-common/src/permissions';
```

Comment:
You can reference `common-common` directly, no need for relative import.

Reply:
yes!